<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Day</class>
 <widget class="QMainWindow" name="Day">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Day</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>761</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>–î–µ–π—Å—Ç–≤–∏—è</string>
    </property>
    <widget class="QPushButton" name="buttonVote">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>–í—ã—Å—Ç–∞–≤–∏—Ç—å –Ω–∞ –≥–æ–ª–æ—Å–æ–≤–∞–Ω–∏–µ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSkip">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>–ü–ê–°</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>–ñ–∞–ª–æ–±–∞ –Ω–∞ –∏–≥—Ä–æ–∫–∞</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonCancel">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>–û—Ç–º–µ–Ω–∞</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>761</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>–†–µ–∞–∫—Ü–∏–∏</string>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>üòí</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>üëé</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>‚ùì</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>üëç</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>‚ùóÔ∏è</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>üíô</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>üíç</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>ü§∑‚Äç</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>üëÆ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>51</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>ü§î</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>–°—Ç–æ–ª</string>
    </property>
    <widget class="QLabel" name="gamer_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>280</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_1">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>280</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>250</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_8">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>210</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>170</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_10">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>261</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>–ò–ù–§–û–†–ú–ê–¶–ò–Ø</string>
     </property>
     <widget class="QLabel" name="labelInformation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>–î–µ–Ω—å –Ω–æ–º–µ—Ä 1</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="gamer_say_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_say_9">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_say_10">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>üòí</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_say_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_say_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>üëç</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_say_1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>130</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>üòí</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_say_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>250</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_say_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>170</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_say_6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>320</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gamer_say_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>320</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelAboutMe">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
